src/lib/pipeline.ml

```diff
@@ -84,9 +87,14 @@ type _ t =
   | Bam_pair: bam t * bam t -> bam_pair t
   | Somatic_variant_caller: somatic Variant_caller.t * bam_pair t -> vcf t
   | Germline_variant_caller: germline Variant_caller.t * bam t -> vcf t
+  | Rna_hla_typer: fastq_sample t -> hla_types t
```

Comment:
I'll address this issue in a future PR when there are actually different 'hla_typers'